CreateMessage/handler.ts

```diff
@@ -416,7 +417,11 @@ export function CreateMessageHandler(
                 organizationFiscalCode: service.organizationFiscalCode,
                 organizationName: service.organizationName,
                 requireSecureChannels: service.requireSecureChannels,
-                serviceCategory: service.serviceMetadata.category,
+                serviceCategory: pipe(
+                  O.fromNullable(service.serviceMetadata),
+                  O.map(_ => _.category),
+                  O.getOrElse(() => StandardServiceCategoryEnum.STANDARD)
+                ),
```

Comment:
```suggestion
                serviceCategory: pipe(
                  O.fromNullable(service.serviceMetadata?.category),
                  O.getOrElse(() => StandardServiceCategoryEnum.STANDARD)
                ),
```
It's the same. This is just a personal flavor
Feel free to take this suggestion